packages/block-library/src/breadcrumbs/index.php

```diff
@@ -172,10 +176,14 @@ function block_core_breadcrumbs_get_hierarchical_post_type_breadcrumbs( $post_id
 	$ancestors        = array_reverse( $ancestors );
 
 	foreach ( $ancestors as $ancestor_id ) {
+		$title = get_the_title( $ancestor_id );
+		if ( strlen( $title ) === 0 ) {
+			$title = __( '(no title)' );
+		}
```

Comment:
Optional: This is a bit repetitive with the above, so we could possibly abstract into a function.